TweetBook.Sdk.Sample/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿
+using Refit;
+using TweetBook.Contracts.V1.Requests;
+using TweetBook.Sdk;
+
+var cachedToken = String.Empty;
+
+var identityApi = RestService.For<IIdentityApi>("https://localhost:7155");
+
+var tweetbookApi = RestService.For<ITweetBookApi>("https://localhost:7155", new RefitSettings
+{
+    AuthorizationHeaderValueGetter = () => Task.FromResult(cachedToken)
+});
+
+var registerResponse = await identityApi.RegisterAsync(new UserRegistrationRequest
+{ 
+    Email = "sdkaccount@gmail.com",
+    Password = "Test1234!"
+});
+
+var loginResponse = await identityApi.LoginAsync(new UserLoginRequest
+{
+    Email = "sdkaccount@gmail.com",
+    Password = "Test1234!"
+});
+
+cachedToken = loginResponse.Content?.Token;
+
+var allPosts = await tweetbookApi.GetAllAsync();
+
+var createdPost = await tweetbookApi.CreateAsync(new CreatePostRequest
+{
+    Name = "This is created by the SDK",
+    Tags = new List<string> { "sdk" }
+});
+
+if (createdPost.Content != null)
+{
+    var retrievedPost = await tweetbookApi.GetAsync(createdPost.Content.Id);
+
+    var updatedPost = await tweetbookApi.UpdateAsync(createdPost.Content.Id, new UpdatePostRequest
+    {
+        Name = "This is updated by the SDK"
+    });
+
+    var deletePost = await tweetbookApi.DeleteAsync(createdPost.Content.Id);
+}

```

Comment:
This file, as a whole has too many empty lines. You may want to delete a few so that it is more readable